mlrun/model_monitoring/helpers.py

```diff
@@ -89,3 +96,49 @@ def get_connection_string(secret_provider: typing.Callable = None) -> str:
         )
         or mlrun.mlconf.model_endpoint_monitoring.endpoint_store_connection
     )
+
+
+def bump_model_endpoint_last_request(
+    project: str,
+    model_endpoint: ModelEndpoint,
+    db: "RunDBInterface",
+    minutes_delta: int = 10,  # TODO: move to config - should be the same as `batch_interval`
+    seconds_delta: int = 2,
+) -> None:
+    """
+    Update the last request field of the model endpoint to be after the current last request time.
+
+    :param project:         Project name.
+    :param model_endpoint:  Model endpoint object.
+    :param db:              DB interface.
+    :param minutes_delta:   Minutes delta to add to the last request time.
+    :param seconds_delta:   Seconds delta to add to the last request time. This is mainly to ensure that the last
```

Comment:
If I understand correctly, I think the 2 seconds adds a risk of skipping the next period:
if the batch-infer was at 9:59, we will set last_request to 20:01 causing us to "skip" the [10,20] period.

Reply:
I think that changing it to 1 second will be ok from both sides.
BTW, I realized that what we really want to bump is "last_updated", so we should add to this delta the 60 seconds wait time